src/Automation/Automation/ChangeLog.md

```diff
@@ -31,6 +31,9 @@
 
 ## Version 1.1.2
 * Fixed issue when retreiving certain monthly schedules in several Azure Automation cmdlets
+* Updated Get-AzAutomationJobOutputRecord to handle JSON and Text record values.
```

Comment:
Can we move this change under the 'Upcoming Release' heading above?  Otherwise, LGTM

Reply:
Just moved to the right spot, not sure how I got it there in the first place.